### ISSUE TITLE
Style me!

### ISSUE THREAD

**Issue by TJkrusinski** [TJkrusinski/orlando-developers#2]
Right now the style sheet is pretty lame, would be great to get a cursory design for the site. I'm thinking something super simple, just enough to cover basic typographic needs.

**Comment by bageera** [comment]
Would bringing bootstrap into the process be too much at this time?

**Comment by TJkrusinski** [comment]
No opposed, however the page really doesn't have any structure, so I'd see it being overkill. I'd love to get more content and things on the page though if you have some ideas.

**Comment by connormckelvey** [comment]
Just fiddling around. I love the typography but hesitant to submit a pull request with the header, because of the HTML required to wrap around it. Furthermore, just with a few lines of HTML and CSS the maintenance time will surely skyrocket. I would suggest a more sophisticated build system with Grunt/Gulp as well as switching to something like Jekyll. I would be more than willing to do this.

![screen shot 2015-03-06 at 1 18 01 pm](https://cloud.githubusercontent.com/assets/262751/6530984/594a9c80-c403-11e4-9631-2712c6216ace.png)

**Comment by TJkrusinski** [comment]
@connormckelvey yeah I agree with all that you're saying. Would love a PR. I'm not tied to metalsmith or the build.sh at all, would love something better.

**Comment by TJkrusinski** [comment]
It'd be great to move the site content out of the markup, somehow.
